src/H5Epublic.h

```diff
@@ -49,14 +49,6 @@ typedef struct H5E_error2_t {
     /**< Optional supplied description      */
 } H5E_error2_t;
 
-/* When this header is included from a private header, don't make calls to H5open() */
```

Comment:
All the copy-and-pasted definitions for the H5OPEN macro have been eliminated and a single one is now in the H5public.h header.